core/cas-server-core-cookie-api/src/test/java/org/apereo/cas/web/support/CookieRetrievingCookieGeneratorTests.java

```diff
@@ -0,0 +1,87 @@
+package org.apereo.cas.web.support;
+
+import org.apereo.cas.authentication.Authentication;
+import org.apereo.cas.authentication.CoreAuthenticationTestUtils;
+import org.apereo.cas.authentication.RememberMeCredential;
+import org.apereo.cas.util.CollectionUtils;
+import org.junit.Test;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.mock.web.MockServletContext;
+import org.springframework.webflow.context.servlet.ServletExternalContext;
+import org.springframework.webflow.test.MockRequestContext;
+
+import javax.servlet.http.Cookie;
+
+import static org.junit.Assert.*;
+
+/**
+ * This is {@link CookieRetrievingCookieGeneratorTests}.
+ *
+ * @author Misagh Moayyed
```

Comment:
This seems like it should be corrected.

Reply:
You mean the name?  I "backported" (aka copy and pasted) this test class from the 6.0 branch, so didn't feel like I should take credit :)